src/components/Viewport.ts

```diff
@@ -1894,9 +1906,9 @@ export default class Viewport {
     };
 
     // Stopped on first cloned first panel
-    if (lastPanelOfPrevDir.getIndex() === 0 && lastPanelOfPrevDir.getCloneIndex() === 0) {
-      newVisibleIndex.min = allPanelCount;
-    }
+    // if (lastPanelOfPrevDir.getIndex() === 0 && lastPanelOfPrevDir.getCloneIndex() === 0) {
```

Comment:
@WoodNeck

Is this code now unnecessary?

Reply:
Yes not they're unnecessary. I've just forgot to remove them.